templates/taxi/manufacturer_list.html

```diff
@@ -23,10 +24,31 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <form action="" method="post">
+                  {% csrf_token %}
+                  <a class="btn btn-primary" href="{% url "taxi:manufacturer-update" pk=manufacturer.id%} " >update</a>
+                </form>
+
+              </td>
+              <td>
+                  <form action="" method="post">
+
+                    {% csrf_token %}
+                    <a class="btn btn-danger" href="{% url "taxi:manufacturer-delete" pk=manufacturer.id%} " >delete</a>
+                  </form>
+
+                </td>
```

Comment:
Indentation is broken here. Also check redundant whitespaces in anchors the same as was mentioned previously.